app/scripts/models/user.js

```diff
@@ -360,8 +360,11 @@ define(function (require, exports, module) {
         .then(function () {
           var isSignUp =
             account.get('verificationReason') === VerificationReasons.SIGN_UP;
+          var emailSent = !! account.get('emailSent');
 
-          if (! account.get('verified') && isSignUp) {
+          // Only send account verification email, it signing up
+          // and a verification email has not been sent by auth-server.
+          if (! account.get('verified') && isSignUp && ! emailSent) {
```

Comment:
Are you going to send the `sendEmailIfUnverified` directly from the fxa-js-client? It'd probably be best to only use the fxa-js-client as a pipe that transfers `sendEmailIfUnverified` from the fxa-js-client consumer to the auth server to avoid making an assumption about who is consuming the client, we don't want to inadvertently break anyone.